_assets/stylesheets/pages/_locations.scss

```diff
@@ -34,26 +34,32 @@ crds-button {
 }
 crds-site-select {
     position: absolute;
-    bottom: 0;
-    left: 125px;
+    // bottom: 0;
+    // left: 125px;
```

Comment:
Kill of commented styles if you don't really need them anymore

Reply:
thanks for the catch! removed